Adds iOS target

esttorhe opened the pull request:
# Fix #24 

I just created a new target with its own test target and "redirected" the same tests to run for the mobile architecture.

A couple of conditional target imports in 3 files and the project now supports iOS framework with tests passing :)

cojoj commented:
@esttorhe I've checked your **PR** and it looks fine but I suggest you change both target names to follow some pattern eg.

- `XcodeServerSDK - OS X`
- `XcodeServerSDK - OS X Tests`
- `XcodeServerSDK - iOS`
- `XcodeServerSDK - iOS Tests`

Also I've noticed that **Tests** for new iOS target are marked as red in _Product_ directory. You know something about this and where to fix this?

When you fix these things I think this **PR** will be ready to merge from my POV.

I've also checked - your **PR** will allow us to use Playground inside project so we can interact with SDK. Everything in that matter is configured as it should be and is described in #26.

---
Good job! 😉 👌

esttorhe commented:
Here's the list of fixes:
1. Renamed targets as suggested on your comment
2. Removed `Info.plist` for `iOS` target as its not needed (using same as the `OSX` target)
3. Renamed product names

---

Regarding your question of the test target in red; basically `Xcode` is a «String based» `IDE`, now that both targets share the same product names `Xcode` will search for the same (in this case `OSX` one) and that's why you see both `iOS` and `OSX` targets in black (because both products are pointing to the same artifact) but the test target had different names and `Xcode` was looking for the `iOS` test product in the same folder as the `OSX` ones failing to find it. ¯\_(ツ)_/¯

czechboy0 commented:
Awesome, thanks @esttorhe and @cojoj. I have some slight edits for this but I'll merge this first and push those changes to the original repo's `swift-2` branch right after.
